src/main.rs

```diff
@@ -82,39 +82,31 @@ struct Opt {
         short = "t",
         long,
         long_help = "\
-            Sets a custom output type:\n\
+            Comma separated list of custom output types:\n\
             - *html* for a HTML coverage report;\n\
             - *coveralls* for the Coveralls specific format;\n\
             - *lcov* for the lcov INFO format;\n\
             - *covdir* for the covdir recursive JSON format;\n\
             - *coveralls+* for the Coveralls specific format with function information;\n\
             - *ade* for the ActiveData-ETL specific format;\n\
             - *files* to only return a list of files.\n\
+            - *cobertura* for output in coberura format.\n\
         ",
         value_name = "OUTPUT TYPE",
         default_value = "lcov",
         requires_ifs = &[
             ("coveralls", "coveralls-auth"),
             ("coveralls+", "coveralls-auth"),
         ],
-        possible_values = &[
-            "ade",
-            "lcov",
-            "coveralls",
-            "coveralls+",
-            "files",
-            "covdir",
-            "html",
-            "cobertura",
-        ],
+        use_delimiter = true
     )]
-    output_type: OutputType,
+    output_types: Vec<OutputType>,
     /// Specifies the output path.
-    #[structopt(short, long, value_name = "PATH", alias = "output-file")]
+    #[structopt(short, long, value_name = "PATH", alias = "output-folder")]
```

Comment:
If I understand correctly, this is either a output path or a output folder, depending on whether multiple output formats are passed or not.

We should then keep the name as it was, so we don't break existing users.

Reply:
pushed